tests/schbench

```diff
@@ -24,4 +23,8 @@ cd $BENCHMARK_ROOT/schbench || die "no $BENCHMARK_ROOT/schbench"
 [ $message_threads -eq 0 ] && message_threads=2
 
 # the source code print the output to stderr, here redirection stderr to stdout
-log_cmd ./schbench -m $message_threads -t $worker_threads -r $runtime -s $sleeptime_us -c $cputime_us 2>&1
+for i in $(seq $iterations)
+do
+	echo Iteration : $i
+	log_cmd ./schbench -m $message_threads -t $worker_threads -r $runtime -s $sleeptime_us -c $cputime_us 2>&1
+done
```

Comment:
does here need do error check in case schbench fails like log_cmd ./schbench || die? As if it fails, the result may be meaningless.

Reply:
Changes are done according to the review comments.